### PR TITLE
feat: updated pydantic version

### PR DESCRIPTION
**Pull request by MrPitonych**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,27 +24,28 @@ keywords = ["taskiq", "tasks", "distributed", "async", "kafka", "aiokafka"]
 packages = [{ include = "taskiq_aio_kafka" }]
 
 [tool.poetry.dependencies]
-python = "^3.8.1"
+python = "^3.9"
 taskiq = "^0"
-aiokafka = "^0.8.0"
-pydantic = "^1.10.7"
+aiokafka = "^0.10.0"
+pydantic = "^2.7.4"
```
Comment (s3rius) [inline pyproject.toml:30]: Let's remove pydantic from dependencies to not enforce a specific verison.

File: taskiq_aio_kafka/broker.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ async def startup(self) -> None:
         self._aiokafka_producer = AIOKafkaProducer(
             bootstrap_servers=self._bootstrap_servers,
             loop=self._loop,
-            **self._aiokafka_producer_params.dict(),
+            **self._aiokafka_producer_params.model_dump(),
```
Comment (chandr-andr) [inline taskiq_aio_kafka/broker.py:152]: It's better to support both v1 and v2 pydantic versions

Comment (s3rius) [inline taskiq_aio_kafka/broker.py:152]: Use `model_dump` function from `taskiq.compat` this will work with any version of pydantic.

**Review by s3rius — CHANGES_REQUESTED** [review]
Let's make this lib pydantic-agnostic. Everything else seems good. Thanks for the PR.

File: taskiq_aio_kafka/broker.py (hunk 2 above)
Comment (MrPitonych) [reply taskiq_aio_kafka/broker.py:152]: ok

Comment (MrPitonych) [reply taskiq_aio_kafka/broker.py]: added model_dump from taskiq.compat

Comment (MrPitonych) [reply taskiq_aio_kafka/broker.py]: > Use `model_dump` function from `taskiq.compat` this will work with any version of pydantic.

check please

**Pull request merged**